mgmt/rest/v2/mock/mock_config_manager.go

```diff
@@ -70,19 +70,25 @@ func (MockConfigManager) DeletePluginConfigDataNodeFieldAll(fields ...string) cd
 // rest_v2_test.go on the plugin config routes found in mgmt/rest/server.go
 const (
 	SET_PLUGIN_CONFIG_ITEM = `{
-  "user": "Jane"
+  "config": {
```

Comment:
@jcooklin, I tried to remove "config" totally. Currently, there is no way to do that in go-swagger. The only way is to add a wrap in snap-client-go to return the response we wanted. Please let me know if you like to do that?